Sources/ETCDExample/ETCDExample.swift

```diff
@@ -13,32 +13,57 @@
 //===----------------------------------------------------------------------===//
 import ETCD
 import NIO
+import Dispatch
```

Comment:
We don't need dispatch here

Reply:
Kept the dispatch since I have a dispatched task

```
DispatchQueue.global().asyncAfter(deadline: .now() + 5) {
                    Task {
                        do {
                            try await etcdClient.set("foo", value: "updated_value")
                        } catch {
                            print("Error setting updated value: \(error)")
                        }
                    }
                }
```

Reply:
You don't need dispatch for this. Just use `Task.sleep(for: .seconds())`